ext/mysql2/extconf.rb

```diff
@@ -66,6 +66,11 @@ def add_ssl_defines(header)
   $LDFLAGS << " -L#{openssl_location}/lib" if openssl_location
 end
 
+if RUBY_PLATFORM =~ /darwin/ && system('command -v brew')
+  zstd_location = `brew --prefix zstd`.strip
+  $LIBPATH << "#{zstd_location}/lib"
```

Comment:
If zstd is not installed (perhaps for an older version of mysqlclient) I think `$LIBPATH` should not be modified. How about this:
```suggestion
  $LIBPATH << "#{zstd_location}/lib" if zstd_location
```